lib/commands/config.js

```diff
@@ -43,10 +43,23 @@ async function config() {
       choices: ['cn-beijing', 'cn-hangzhou', 'cn-shanghai', 'cn-shenzhen' , 'cn-hongkong',
         'ap-southeast-1', 'ap-southeast-2', 'ap-northeast-1', 'eu-central-1'],
       default: profile.defaultRegion
+    },
+    {
+      type: 'input',
+      name: 'timeout',
+      message: 'client timeout in seconds',
```

Comment:
The timeout in seconds for each SDK client invoking

Reply:
done